src/components/app-shell.tsx

```diff
@@ -15,11 +15,13 @@ export default class AppShell extends React.Component<Props> {
   public render() {
     return (
       <div>
-        <Route path='/' component={App} />
-        <Route path='/edited' component={App} />
-        <Route path='/gist/:mode/:username/:id' component={App} />
-        <Route path='/examples/:mode/:example_name' component={App} />
-        <Route path='/custom/:mode' component={App} />
+        <Switch>
+          <Route path='/' exact component={App} />
```

Comment:
Is exact still needed here now that we use switch?

Reply:
Without `exact`, all paths will always mount the first `<Route />` since every path begins with `/`.